### PR TITLE
Tag rewrites for the new registerTag format.

### PR DESCRIPTION
**Pull request by Talamar1**

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizencore/objects/dList.java [hunk 1]
```diff
@@ -319,45 +319,76 @@ public static void registerTags() {
...
+        registerTag("separated_by", new TagRunnable() {
+            @Override
+            public String run(Attribute attribute, dObject object) {
+                dB.log("inside of tag register");
+                dList list = (dList) object;
+                if (list.isEmpty()) {
+                    return new Element("").getAttribute(attribute.fulfill(1));
+                }
+                String input = attribute.getContext(1);
+                dB.log("input:" + input);
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizencore/objects/dList.java]: Excuse you?

File: src/main/java/net/aufdemrand/denizencore/objects/dList.java [hunk 2]
```diff
@@ -631,31 +597,38 @@ public String getAttribute(Attribute attribute) {
...
+                    result.remove(index);
+                    for (int i = 0; i < items.size(); i++) {
+                        result.add(index + i, items.get(i));
+                    }
+                    return result.getAttribute(attribute.fulfill(1));
+                }
+                else {
+                    dB.echoError("The tag li@list.set[...] must be followed by .at[#]!");
+                    return null;
+                }            }
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizencore/objects/dList.java]: dafuq

File: src/main/java/net/aufdemrand/denizencore/objects/dList.java (hunk 1 above)
Comment (Talamar1) [inline src/main/java/net/aufdemrand/denizencore/objects/dList.java]: I removed that in the followup commit already.

File: src/main/java/net/aufdemrand/denizencore/objects/dList.java (hunk 2 above)
Comment (Talamar1) [inline src/main/java/net/aufdemrand/denizencore/objects/dList.java]: I blame IntelliJ

Comment (Talamar1) [inline src/main/java/net/aufdemrand/denizencore/objects/dList.java]: Fixed.

**Pull request merged**